### PR TITLE
Added steps to load extension

### PR DESCRIPTION
**Pull request by RishabhJain0721**

### CODE REVIEW

**Comment by chaitanyabisht** [comment]
It's better to let the user pull the extension instead of downloading the zip. For the end user who just wants to use the extension, an option of downloading the .crx file is provided.

For now just change from downloading zip file to pull using SSH/HTTPS.

**Comment by RishabhJain0721** [comment]
Actually I have tried cloning this repo using HTTPS but there is some kind of error due to which only **.git** folder is being cloned . And it's happening only with this repo; others are being cloned successfully. 

That's why I added "downloading zip" step.

error

**Comment by chaitanyabisht** [comment]
I think it's something specific to your machine, I have cloned the repo myself using SSH and HTTPS too and it works. Try using SSH maybe?

**Comment by RishabhJain0721** [comment]
@chaitanyabisht I have made the required changes.

**Comment by Ash-KODES** [comment]
> Actually I have tried cloning this repo using HTTPS but there is some kind of error due to which only **.git** folder is being cloned . And it's happening only with this repo; others are being cloned successfully.
> 
> That's why I added "downloading zip" step.
> 
> error

Actually I got the same issue on my window's machine but cloning worked completely fine on my ubuntu's machine.

**Comment by chaitanyabisht** [comment]
I think it's something related to extension/socket.io.js.map:Zone.Identifier file. It seems that Windows may not be allowing certain characters in this file name, hence not allowing the cloning of the repo. I'll see and remove this file from the repo as I think this should not be the part of repo.

I myself worked on Linux and hence never faced this issue.

File: README.md [hunk 1]
```diff
@@ -17,10 +17,57 @@ All users open a shared link to a YouTube video and then join the room. Please n
...
+ 
+ ### Your extension is successfully loaded
+ ### It should look like this :  
+ ![Screenshot 2022-12-16 154116](https://user-images.githubusercontent.com/107767172/208075717-329ff658-b6a4-4664-9aba-e8de20c07cbd.png)
+
+
+### To start the server locally : 
+
+ 1. Open the "server" folder that we downloaded earlier.
+ 2. Open command line there and run this command :    **node index.js**
```
Comment (chaitanyabisht) [inline README.md]: Add npm install before this as you need to install the packages

Comment (chaitanyabisht) [reply README.md]: I meant just add the command "npm install" so that we install all the dependencies

File: README.md [hunk 2]
```diff
@@ -62,8 +62,10 @@ The video will sync automatically when the host plays/pauses the video or seeks
 
 ### To start the server locally : 
 
- 1. Open the "server" folder that we downloaded earlier.
- 2. Open command line there and run this command :    **node index.js**  
+ 1. Install node on your local from this link :  https://nodejs.org/en/download/
+ 2. Open command line and install npm using this command : **npm install -g npm**
```
Comment (chaitanyabisht) [inline README.md]: I meant just add the command "npm install" so that we install all the dependencies

Comment (RishabhJain0721) [reply README.md]: I have made the changes..Please review @chaitanyabisht

File: README.md [hunk 3]
```diff
@@ -62,10 +62,9 @@ The video will sync automatically when the host plays/pauses the video or seeks
 
 ### To start the server locally : 
 
- 1. Install node on your local from this link :  https://nodejs.org/en/download/
- 2. Open command line and install npm using this command : **npm install -g npm**
- 3. Open the "server" folder that we downloaded earlier.
- 4. Open command line there and run this command :    **node index.js**  
+ 1. Open the "server" folder and open command line there.
+ 2. Run this command to install dependencies : **npm i cors ejs express socket.io**
```
Comment (chaitanyabisht) [inline README.md]: You just need to write "npm install", package.json file has all the dependencies which are necessary to be installed

File: README.md [hunk 4]
```diff
@@ -17,10 +17,59 @@ All users open a shared link to a YouTube video and then join the room. Please n
...
-<a href="https://imgbb.com/"><img src="https://i.ibb.co/mc8d4fh/Screenshot-from-2022-12-13-18-05-34.png" alt="Screenshot-from-2022-12-13-18-05-34" border="0" /></a>
+<a href="https://imgbb.com/"><img src="https://i.ibb.co/mc8d4fh/Screenshot-from-2022-12-13-18-05-34.png" alt="Screenshot-from-2022-12-13-18-05-34" border="0" width=30% /></a>
 
 ### Chat feature
-<a href="https://imgbb.com/"><img src="https://i.ibb.co/0VP476c/Screenshot-from-2022-12-13-18-09-37.png" alt="Screenshot-from-2022-12-13-18-09-37" border="0" /></a>
+<a href="https://imgbb.com/"><img src="https://i.ibb.co/0VP476c/Screenshot-from-2022-12-13-18-09-37.png" alt="Screenshot-from-2022-12-13-18-09-37" border="0" width=30%/></a>
+
+## 🔧 How to get the extension loaded?
+### Steps : 
+1. Copy the HTTPS url of the repository.
```
Comment (chaitanyabisht) [inline README.md]: Make it HTTPS/SSH

**Pull request merged**